docs/assistant/ai-esql-queries.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[esql-queries-assistant]]
+= Generate, customize, and learn about {esql} queries
+
+:frontmatter-description: Elastic AI Assistant can help you write ES|QL queries.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [guide]
+:frontmatter-tags-user-goals: [get-started]
+
+This page describes some of the ways Elastic AI Assistant can help you leverage the Elasticsearch Query Language ({esql}). 
```

Comment:
It feels a little unnecessary for the page to announce itself in this way. (I know that works in other situations, so I'm not sure why it doesn't work for me here. Maybe because it's such a simple, direct statement, and the extra phrasing dilutes its effect?)
```suggestion
Elastic AI Assistant can help you learn about and leverage the Elasticsearch Query Language ({esql}). 
```